aspnetcore/security/authentication/google-logins.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Google external login setup
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 11/2/2016
+ms.topic: article
+ms.assetid: 8BA389D6-0911-4415-A818-C7B3F5B5CC8D
+ms.prod: aspnet-core
+﻿uid: security/authentication/google-logins
+---
+# Configuring Google authentication
+
+<a name=security-authentication-google-logins></a>
+
+This tutorial shows you how to enable your users to login with their Google+ account using a sample ASP.NET Core project created in the [previous section](sociallogins.md). We start by following the [official steps](https://developers.google.com/identity/sign-in/web/devconsole-project) to create a new app in Google API Console.
+
+## Creating the app in Google API Console
+
+* Navigate to [https://console.developers.google.com/projectselector/apis/library](https://console.developers.google.com/projectselector/apis/library) and sign in. If you don't already have a Google account, use the *[Create account](https://accounts.google.com/SignUpWithoutGmail?service=cloudconsole&continue=https%3A%2F%2Fconsole.developers.google.com%2Fprojectselector%2Fapis%2Flibrary&ltmpl=api)* link to create one:
+
+![image](sociallogins/_static/GoogleConsoleLogin.png)
+
+* You will be redirected to API Manager Library page:
+
+![image](sociallogins/_static/GoogleConsoleSwitchboard.png)
+
+* Tap **Create a project** and enter your application name:
+
+![image](sociallogins/_static/GoogleConsoleNewProj.png)
+
+* You will be redirected back to the Library page allowing you to choose features your new app will support. Find *Google+ API* in the list and click on its link to add the API feature:
```

Comment:
After going through the process with Google+ left disabled, the middleware appears to have gotten 403 Forbidden from Google. I assume this is what you meant.